safe_autonomy_sims/gym/inspection/inspection_v0.py

```diff
@@ -267,9 +270,10 @@ def step(
 
         # Store previous simulator state
         self.prev_state = self.sim_state.copy()
-        self.prev_num_inspected = (
-            self.chief.inspection_points.get_num_points_inspected()
-        )
+        if self.simulator.sim_time > 0:
+            self.prev_num_inspected = (
+                self.chief.inspection_points.get_num_points_inspected()
+            )
```

Comment:
This will not account for points seen on initialization which I believe is a valid condition. Is this intentional? What behavior does the CoRL version exhibit?

Reply:
Interestingly, this has the opposite effect. Since the `prev_num_inspected` attribute is used in the calculation of the inspected points reward [here](https://github.com/act3-ace/safe-autonomy-sims/blob/main/safe_autonomy_sims/gym/inspection/inspection_v0.py#L335) and [here](https://github.com/act3-ace/safe-autonomy-sims/blob/main/safe_autonomy_sims/gym/inspection/reward.py#L33), the points inspected during environment reset are never counted! The flow is: reset gets called causing some initial points to be inspected, the first step gets called, previous points inspected attribute gets assigned (the num inspected points FROM reset), then the simulator is stepped, then inspected points reward is calculated. 

This conditional keeps the previous_num_inspected attribute at 0 for the first step of each episode.

Reply:
The CoRL inspection environments all return rewards for the points inspected during initialization.